docs/crio.8.md

```diff
@@ -5,160 +5,128 @@
+[--root=[value]]                   \
+[--runroot=[value]]                \
+[--runtime=[value]]                \
+[--seccomp-profile=[value]]        \
+[--selinux]                        \
+[--signature-policy=[value]]       \
+[--storage-driver=[value]]         \
+[--storage-opt=[value]]            \
+[--version|-v]          
+```
 # DESCRIPTION
 OCI-based implementation of Kubernetes Container Runtime Interface Daemon
 
 crio is meant to provide an integration path between OCI conformant runtimes and the kubelet. Specifically, it implements the Kubelet Container Runtime Interface (CRI) using OCI conformant runtimes. The scope of crio is tied to the scope of the CRI.
 
-	* Support multiple image formats including the existing Docker image format
-	* Support for multiple means to download images including trust & image verification
-	* Container image management (managing image layers, overlay filesystems, etc)
-	* Container process lifecycle management
-	* Monitoring and logging required to satisfy the CRI
-	* Resource isolation as required by the CRI
-
-**crio [GLOBAL OPTIONS]**
-
-**crio [GLOBAL OPTIONS] config [OPTIONS]**
-
+- Support multiple image formats including the existing Docker image format
+- Support for multiple means to download images including trust & image verification
+- Container image management (managing image layers, overlay filesystems, etc)
+- Container process lifecycle management
+- Monitoring and logging required to satisfy the CRI
+- Resource isolation as required by the CRI
+
+Usage: 
+```
+crio [GLOBAL OPTIONS]
+crio [GLOBAL OPTIONS] config [OPTIONS]
+```
 # GLOBAL OPTIONS
 
-**--apparmor_profile**=""
-  Name of the apparmor profile to be used as the runtime's default (default: "crio-default")
+- **--apparmor_profile**="": Name of the apparmor profile to be used as the runtime's default (default: "crio-default")
 
-**--cgroup-manager**=""
-  cgroup manager (cgroupfs or systemd)
+- **--cgroup-manager**="": cgroup manager (cgroupfs or systemd)
 
-**--config**=""
-  path to configuration file
+- **--config**="": path to configuration file
 
-**--conmon**=""
-  path to the conmon executable (default: "/usr/local/libexec/crio/conmon")
+- **--conmon**="": path to the conmon executable (default: "/usr/local/libexec/crio/conmon")
 
-**--cpu-profile**=""
-set the CPU profile file path
+- **--cpu-profile**="": set the CPU profile file path
 
-**--default-transport**
-  A prefix to prepend to image names that can't be pulled as-is.
+- **--default-transport**: A prefix to prepend to image names that can't be pulled as-is.
 
-**--help, -h**
-  Print usage statement
+- **--help, -h**: Print usage statement
 
-**--insecure-registry=**
-  Enable insecure registry  communication,  i.e.,  enable  un-encrypted
+- **--insecure-registry=**: Enable insecure registry  communication,  i.e.,  enable  un-encrypted
   and/or untrusted communication.
+  > List  of  insecure registries can contain an element with CIDR notation
```

Comment:
This change  puts these lines in a new font.  Can you just make these a bullet list as you did image-volumes below?